docs/src/main/asciidoc/sqs.adoc

```diff
@@ -611,6 +611,19 @@ Any number of `@SqsListener` annotations can be used in a bean class, and each a
 
 NOTE: Queues declared in the same annotation will share the container, though each will have separate throughput and acknowledgement controls.
 
+===== SNS Messages
+
+When receiving SNS messages through the `@SqsListener`, the message includes all attributes of the `SnsNotification`. To only receive need the `Message` part of the payload, you can utilize the `@SnsNotificationMessage` annotation.
+[source, java]
+----
+@SqsListener("my-queue")
+public void listen(@SnsNotificationMessage Pojo pojo) {
+	System.out.println(pojo.field);
+}
+----
+
+NOTE: You will need to configure an `ObjectMapper` with `SqsListenerConfigurer` in order to utilize this annotation.
```

Comment:
We can leave this note out - in a regular Spring Boot application an `ObjectMapper` will already be auto configured.